.gitmodules

```diff
@@ -12,7 +12,7 @@
 	url = https://github.com/robbiehanson/CocoaHTTPServer.git
 [submodule "lib/PublicAutomation"]
 	path = lib/PublicAutomation
-	url = https://github.com/TestingWithFrank/PublicAutomation.git
```

Comment:
I've pulled in your PublicAutomation fix. Please restore this submodule to point back to the `TestingWithFrank/PublicAutomation` repo